airbyte-ci/connectors/connector_ops/tests/test_utils.py

```diff
@@ -105,49 +123,3 @@ def test_parse_dependencies(gradle_file_with_dependencies):
     assert all([regular_dependency in expected_regular_dependencies for regular_dependency in regular_dependencies])
     assert len(test_dependencies) == len(expected_test_dependencies)
     assert all([test_dependency in expected_test_dependencies for test_dependency in test_dependencies])
-
-
-@pytest.mark.parametrize("with_test_dependencies", [True, False])
-def test_get_all_gradle_dependencies(with_test_dependencies):
-    build_file = Path("airbyte-integrations/connectors/source-postgres-strict-encrypt/build.gradle")
-    if with_test_dependencies:
-        all_dependencies = utils.get_all_gradle_dependencies(build_file)
-        expected_dependencies = [
-            Path("airbyte-cdk/java/airbyte-cdk"),
-            Path("airbyte-db/db-lib"),
-            Path("airbyte-json-validation"),
-            Path("airbyte-config-oss/config-models-oss"),
-            Path("airbyte-commons"),
-            Path("airbyte-test-utils"),
-            Path("airbyte-api"),
-            Path("airbyte-connector-test-harnesses/acceptance-test-harness"),
-            Path("airbyte-commons-protocol"),
-            Path("airbyte-integrations/bases/base-java"),
-            Path("airbyte-commons-cli"),
-            Path("airbyte-integrations/bases/base"),
-            Path("airbyte-integrations/connectors/source-postgres"),
-            Path("airbyte-integrations/bases/debezium"),
-            Path("airbyte-integrations/connectors/source-jdbc"),
-            Path("airbyte-integrations/connectors/source-relational-db"),
-            Path("airbyte-integrations/bases/standard-source-test"),
-        ]
-        assert len(all_dependencies) == len(expected_dependencies)
-        assert all([dependency in expected_dependencies for dependency in all_dependencies])
-    else:
-        all_dependencies = utils.get_all_gradle_dependencies(build_file, with_test_dependencies=False)
-        expected_dependencies = [
-            Path("airbyte-cdk/java/airbyte-cdk"),
-            Path("airbyte-db/db-lib"),
-            Path("airbyte-json-validation"),
-            Path("airbyte-config-oss/config-models-oss"),
-            Path("airbyte-commons"),
-            Path("airbyte-integrations/bases/base-java"),
-            Path("airbyte-commons-cli"),
-            Path("airbyte-integrations/bases/base"),
-            Path("airbyte-integrations/connectors/source-postgres"),
-            Path("airbyte-integrations/bases/debezium"),
-            Path("airbyte-integrations/connectors/source-jdbc"),
-            Path("airbyte-integrations/connectors/source-relational-db"),
-        ]
-        assert len(all_dependencies) == len(expected_dependencies)
-        assert all([dependency in expected_dependencies for dependency in all_dependencies])
```

Comment:
This test is brittle as it's vulnerable to connector specific changes. It does pass anymore as the source-postgres-strict-encrypt connector dependencies changed.